src/int/specialized_div_rem/mod.rs

```diff
@@ -300,6 +300,8 @@ impl_asymmetric!(
 );
 
 // 32 bits is the smallest division used by `compiler-builtins`, so we end with binary long division
+// FIXME: remove dead_code
+#[allow(dead_code)]
```

Comment:
This fixme for https://github.com/rust-lang/compiler-builtins/actions/runs/5786737992/job/15682137705#step:8:330
```
   --> src/int/specialized_div_rem/mod.rs:304:5
    |
304 |     u32_div_rem,
    |     ^^^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default
    = note: this warning originates in the macro `impl_binary_long` (in Nightly builds, run with -Z macro-backtrace for more info)
```
how this should be gated?

Reply:
Attributes don't work on macros. You need to put the `#[allow]` on the actual function inside the macro definition.

Reply:
Yes, i see warning now, but why it prevents CI fail in that case?